dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/AbstractTeradataConnector.java

```diff
@@ -87,6 +93,31 @@ public abstract class AbstractTeradataConnector extends AbstractJdbcConnector {
   @VisibleForTesting /* pp */ static final String DEF_LOG_TABLE = "dbc.DBQLogTbl";
   @VisibleForTesting /* pp */ static final String DEF_QUERY_TABLE = "dbc.DBQLSQLTbl";
 
+  protected enum CommonTeradataConnectorProperty implements ConnectorProperty {
+    TMODE(
+        "teradata.tmode",
+        "Transaction mode - one of ANSI, TERA, DEFAULT or NONE. If NONE is used, then no"
```

Comment:
What's the difference between using `NONE` vs `DEFAULT`?

Reply:
ref: https://teradata-docs.s3.amazonaws.com/doc/connectivity/jdbc/reference/current/jdbcug_chapter_2.html#URL_TMODE

Reply:
`NONE` does not specify the parameter in the JDBC connection string. `DEFAULT` passes the value `DEFAULT` to the Teradata server. It's possible that the effect is the same, but I wanted to keep both options for completeness, in case there is some difference.